package.json

```diff
@@ -5,8 +5,10 @@
   "main": "build/es5/index.js",
   "types": "build/typings/index.d.ts",
   "scripts": {
+    "build": "grunt",
     "debug-mocha": "iron-node node_modules/mocha/bin/_mocha --require mochahook",
-    "test": "mocha test/unit --require mochahook",
+    "test-unit": "mocha test/unit --require mochahook",
```

Comment:
Could we keep "test" script for unit tests and use "test-functional" for functional testing? Just to keep script actions as they were before this change and for consistency with other projects. Most of developers expect npm run test to be unit testing.

Reply:
test stage of travis is the main problem, it starts the test script defined in the package.json. I have to discuss this point with @bbert but I think that we tried to call a specific test script in the travis.yml : it didn't work. :-(

Reply:
Ok. Please, let me know if it finally worked. Otherwise we can do as you proposed.